macOSUpgrade.sh

```diff
@@ -170,6 +198,41 @@ downloadInstaller() {
     /bin/kill "${jamfHUDPID}"
 }
 
+get_power_status() {
```

Comment:
When I saw the process, I put errors into variables. Therefore, the naming 'get' seems to be inappropriate.
For example, how about naming such as 'validate'?

Reply:
Replaced get_ with validate_.
View [3c0067c](https://github.com/kc9wwh/macOSUpgrade/pull/115/commits/3c0067c1bbf88e2ab4059688ad4c12a183d5dc85)